lua/nvim-treesitter/parsers.lua

```diff
@@ -2004,6 +2004,7 @@ list.t32 = {
     url = "https://gitlab.com/xasc/tree-sitter-t32.git",
     files = { "src/parser.c", "src/scanner.c" },
   },
+  filetype = "trace32",
```

Comment:
This means it will no longer start for a `t32` filetype.

Reply:
Vimland doesn't have t32 as a filetype anymore so I don't think t32 fieltype is needed

Reply:
Just wanted to make sure that's intended.

Reply:
I don't think Vim really had a file type `t32` in the past, so this change shouldn't cause many problems for users.

The only complaints I received were that the new file type may break custom file type definitions in Vim and that it was not immediately obvious what had been causing this.